test/test_queue.js

```diff
@@ -318,6 +318,49 @@ describe('Queue', function () {
       });
     });
 
+    it('should processes jobs by priority', function(done){
+      var normalPriority = [],
+        mediumPriority = [],
+        highPriority = [];
+
+      // for the current strategy this number should not exceed 8 (2^2*2)
+      // this is done to maitain a deterministic output.
```

Comment:
This comment does not hold anymore since we do not support any strategies (yet).